### DISCUSSION TITLE
Jupyter Notebook

### DISCUSSION THREAD

**Discussion by R-Rob** [cooperrc/engineering-dynamics#3] [Q&A]
Hello, would anyone be willing to walk me through how to turn in the Jupyter Notebook for HW0? Thanks!

**Comment by aidanfurlonge** [comment]
Once you are done editing the notebook hit ctrl+p or command+p depending on your operating system, and select Save as PDF. Then on HuskyCT go to 'Module_00' and open the 'HW_00' folder. Then go to 'Submit HW_00' and you can upload your PDF there. Hope this helps.

**Reply by cooperrc** [reply]
One other step is:
1. Click File -> print preview
2. ctr-p (apple-p)
3. save as pdf
